### ISSUE TITLE
Allow to define number of available threads on @Actor annotation

### ISSUE THREAD

**Issue by dabla** [wblancqu/actress#2]
It would be nice to allow one Actor to run on multiple threads, or does this break the concept?  You could implement this by defining a semaphore for each Actor like I did with ParallelIterable...

**Comment by wblancqu** [comment]
Correct, started an implementation of this. Can be useful in systems having lots of actors. Provide an "ActorPool" from which Actors can borrow threads. However the same Actor can never be active on multiple threads in parallel, but he can be active on Thread-1 now, and on Thread-2 in 10 seconds.

**Comment by dabla** [comment]
Cool! Looking forward to it! ;-)

**Comment by dabla** [comment]
Maybe you could also forsee a second attribute allowing you the customize the thread name prefix which is always handy when logging messages ;-)

**Comment by wblancqu** [comment]
How about @Actor(pool = CPU), @Actor(pool = IO), and @Actor(pool = SINGLE_THREAD) as a default?

**Comment by dabla** [comment]
Maybe rename it to poolStrategy? Values are fine! And add a second attribute named poolNumber to be able to fine tune it if you don't want to use predefined strategies? And not to forget threadNamePattern for logging purposes 😉

**Comment by vanhove** [comment]
A configurableInterface would be nice to, in the situation that you want to set the value from a property file or spring context.
